R/install.R

```diff
@@ -334,19 +334,19 @@
 #' @return `BiocManager::install()` returns the `pkgs` argument, invisibly.
 #' @seealso
 #'
-#' `BiocManager::\link{repositories}()` returns the _Bioconductor_ and
+#' [`BiocManager::repositories()`] returns the _Bioconductor_ and
```

Comment:
I think this and similar, below, would be better as `` `BiocManager::repositories()` ``, without the link, following the logic in the comment above. Alternatively, I think they should be formatted with the fully resolved function `BiocManager::repositories()`, and with 'repositories' linked to the help page -- old-school `\code{BiocManager::\link{repositories}()}` I think.